src/Tickets/Commerce/Gateways/Stripe/Webhooks.php

```diff
@@ -89,6 +98,56 @@ public function get_merchant(): Abstract_Merchant {
 		return tribe( Merchant::class );
 	}
 
+	/**
+	 * Add a pending webhook to the order.
+	 *
+	 * @since TBD
+	 *
+	 * @param int    $order_id   Order ID.
+	 * @param string $new_status New status.
+	 * @param string $old_status Old status.
+	 * @param array  $metadata   Metadata.
+	 *
+	 * @return void
+	 */
+	public function add_pending_webhook( int $order_id, string $new_status, string $old_status, array $metadata = [] ): void {
+		add_post_meta(
+			$order_id,
+			self::PENDING_WEBHOOKS_KEY,
+			[
+				'new_status' => $new_status,
+				'metadata'   => $metadata,
+				'old_status' => $old_status,
+			]
+		);
+	}
+
+	/**
+	 * Get the pending webhooks for an order.
+	 *
+	 * @since TBD
+	 *
+	 * @param int $order_id Order ID.
+	 *
+	 * @return array
+	 */
+	public function get_pending_webhooks( int $order_id ): array {
+		return (array) get_post_meta( $order_id, self::PENDING_WEBHOOKS_KEY );
+	}
+
+	/**
+	 * Delete the pending webhooks for an order.
+	 *
+	 * @since TBD
+	 *
+	 * @param int $order_id Order ID.
+	 *
+	 * @return void
+	 */
+	public function delete_pending_webhooks( int $order_id ): void {
+		delete_post_meta( $order_id, self::PENDING_WEBHOOKS_KEY );
```

Comment:
why not return if the deletion was successful?

Reply:
Because currently we dont care - we are not going to do anything differently succeeded or failed.